pkg/supportbundle/collect.go

```diff
@@ -253,3 +217,132 @@ func getAnalysisFile(analyzeResults []*analyze.AnalyzeResult) (io.Reader, error)
 
 	return bytes.NewBuffer(analysis), nil
 }
+
+// collectRemoteHost runs remote host collectors sequentially
+func collectRemoteHost(ctx context.Context, filteredCollectors []FilteredCollector, bundlePath string, opts SupportBundleCreateOpts, collectedData map[string][]byte) error {
+	// Run remote collectors sequentially
+	for _, collectorData := range filteredCollectors {
+		collector := collectorData.Collector
+		spec := collectorData.Spec
+
+		// Send progress event: starting the collector
+		opts.ProgressChan <- fmt.Sprintf("[%s] Running host collector...", collector.Title())
+
+		// Start a span for tracing
+		_, span := otel.Tracer(constants.LIB_TRACER_NAME).Start(ctx, collector.Title())
+		span.SetAttributes(attribute.String("type", reflect.TypeOf(collector).String()))
+
+		// Parameters for remote collection
+		params := &collect.RemoteCollectParams{
+			ProgressChan:  opts.ProgressChan,
+			HostCollector: &spec,
+			BundlePath:    bundlePath,
+			ClientConfig:  opts.KubernetesRestConfig,
+			Image:         "replicated/troubleshoot:latest",
+			PullPolicy:    "IfNotPresent",
+			Timeout:       time.Duration(60 * time.Second),
+			LabelSelector: "",
+			NamePrefix:    "hostos-remote",
```

Comment:
should it be renamed to host-remote, since it has been moved into a general function?